src/main/java/org/folio/oaipmh/Request.java

```diff
@@ -266,6 +268,27 @@ public boolean isResumptionTokenParsableAndValid() {
     return true;
   }
 
+  public boolean isResumptionTokenTimeExpired() {
+    try {
+      String resumptionToken = new String(Base64.getUrlDecoder().decode(oaiRequest.getResumptionToken()),
+        StandardCharsets.UTF_8);
+
+      Map<String, String> params;
+      params = URLEncodedUtils
```

Comment:
Please fix misprint Map<String, String> params = and removing Map<String, String> params;